### DISCUSSION TITLE
scrum prep Milestone 2 - 2

### DISCUSSION THREAD

**Discussion by MrL9000** [uprm-inso4116-2024-2025-s1/semester-project-trolley-tracker-app#112] [General]
Write your scrum here.

**Comment by MrL9000** [comment]
At the start of this week I started programing the menu for the vheicle tab, unfortunately I didn't get a chance to work on the project any further, that said, I did do tasks in the first Sprint for the milestone, those being the ones where I established where the codebase for our team is and how we would push our code via communicating with the Manager and Team Leader.

edit: I also did a lecture topic task where I found certain implied concepts that weren't being made clear in the model of the project.

**Comment by AngelMartinez33** [comment]
Had team weekly meeting, created a graphic detailing documentation instructions, created and supervised blue team tasks, and ensured the team is going forward.

**Comment by warm007** [comment]
During this week, I attended the general meeting and team meetings. I worked on a documentation task, specifically (task #117) Revise 1.2.1 Current Situation and Fix 1.2.2 Needs based on professor feedback for Milestone 1. However, it was a busy week with other classes, so I was only able to complete the documentation task. I am also progressing on two development tasks: Selected Ownership View Creation (task #86) and Vehicle Information Refactoring (task #75). I expect to complete the vehicle refactoring task over the weekend.

**Comment by Janiel777** [comment]
Durante la semana cree un task para que se lo asignara un miembro de mi equipo y creé uno para mí. Ambos task fueron completados como era esperado. Además asistí a la reunión que realizan los managers semanalmente.

Los tasks fueron:
https://github.com/uprm-inso4116-2024-2025-s1/semester-project-trolley-tracker-app/issues/116

https://github.com/uprm-inso4116-2024-2025-s1/semester-project-trolley-tracker-app/issues/104

**Comment by AlexisTraversoPR** [comment]
This week I implemented a function that removes notifications from the notification side menu at the click of a button. I also properly implemented a fix discussed with the manager last week to the order of the notifications so that newer notifications show up at the top of the side menu.

Tasks completed:
#99 
#76

**Comment by JaseyAcevedo** [comment]
Been dealing with exams and projects this week.

**Comment by Brandell13** [comment]
This week I was working on two tasks, one on LTT and the other on application code. Additionally, I was meeting with the team to see the group's progress on the project.

**Comment by Alej24** [comment]
During this week I attended the team meetings and completed one task that changed the angular tab icon to the RUMBus logo.  Due to this being a busy week, I haven't managed to finish another task.

**Comment by diegorobles2** [comment]
This week I was working on the proyect tasks.

**Comment by AleDiazz** [comment]
attended meeting, kept working on my coding task and finished one of my two lecture topic task for this milstone

**Comment by Yahid1** [comment]
This week, I coordinated and clarified all tasks as well as ensured task creation for milestone objectives. Closed and reviewed tasks as well as helped people understand the code. Backend tutorials are available and posted.

**Comment by gabrielpadilla7** [comment]
Reviewed tasks and closed them. Monitored changes in the documentation, as well as helping the team follow guidelines and possible changes.

**Comment by yamayrie** [comment]
Attended weekly general meeting, created a lecture topic task that was added to documentation. All info about the lecture topic task can be found on issue #109.

**Comment by marcosarraiza** [comment]
Attended our weekly meetings and created an Ownership View for our app, this was in issue #85.

**Comment by sebasagir** [comment]
this week I mainly worked on the task of showing the backend notifications instead of using dummy data

**Comment by rafa-quiles** [comment]
This week I attended meetings. And worked on my tasks

**Comment by juan-ortiz92** [comment]
This week I have been studying and reviewing to work on my next lecture topic task.

**Comment by JoelTorres20** [comment]
This week I started working on my current opened task and additionally I also started and finished my first lecture topic task.

**Comment by dylanDeJesus21** [comment]
Created lecture Topic task and studied RumBus code. Attended team meeting

**Comment by IvanierB** [comment]
This week I had a hell week with 3 test and didn't had a chance to work with the project.

**Comment by dAninAd1248** [comment]
I was too busy catching up to lectures I missed to do anything else.

**Comment by JConcepcion01** [comment]
So I may have forgotten to post here by last by Friday, but throughout that week I worked got assigned some tasks and worked on of them, mostly a more simple one to work on some of the documentation readme file for the project

**Comment by dylanDeJesus21** [comment]
This week I was able to give each entity in this project a proper description as well as reletionships between them. Aside from that I eliminated section 1.4 from the documentation and integrate its into the rest of the sections
